app/src/main/java/com/liato/bankdroid/BankdroidApplication.java

```diff
@@ -24,22 +22,16 @@
     @Override
     public void onCreate() {
         super.onCreate();
-        Fabric.with(this, new Crashlytics(), new Answers());
-        Crashlytics.setString(LOG_KEY_LOCALE, Locale.getDefault().toString());
-        logBanks();
-    }
 
-    private void logBanks() {
-        List<Bank> banks = BankFactory.banksFromDb(this, false);
-        StringBuilder bankStringBuilder = new StringBuilder();
-        if (banks != null && !banks.isEmpty()) {
-            for (Bank bank : banks) {
-                bankStringBuilder.append(bank.getName())
-                        .append(",");
-            }
-            Crashlytics.setString(LOG_KEY_BANKS,
-                    bankStringBuilder.substring(0, bankStringBuilder.length() - 1));
+        CrashlyticsCore core = new CrashlyticsCore.Builder()
+                .disabled(BuildConfig.DEBUG)
+                .build();
+        Fabric.with(this, new Crashlytics.Builder().core(core).build());
+
+        if (BuildConfig.DEBUG) {
+            Timber.plant(new Timber.DebugTree());
         }
+        Timber.plant(new CrashlyticsTree());
```

Comment:
Thanks for the use case. I will verify the widget behaviour with `Application#onCreate()`